doc/debian/jitsi-meet-prosody/prosody.cfg.lua-jvb.example

```diff
@@ -6,9 +6,9 @@ muc_mapper_domain_base = "jitmeet.example.com";
 turncredentials_secret = "__turnSecret__";
 
 turncredentials = {
-  { type = "stun", host = "jitmeet.example.com", port = "4446" },
-  { type = "turn", host = "jitmeet.example.com", port = "4446", transport = "udp" },
-  { type = "turns", host = "jitmeet.example.com", port = "443", transport = "tcp" }
+  { type = "stun", host = "jitmeet.example.com", port = "3478" },
+  { type = "turn", host = "jitmeet.example.com", port = "3478", transport = "udp" },
+  { type = "turns", host = "jitmeet.example.com", port = "5349", transport = "tcp" }
```

Comment:
You should not change 443 here in the current setup. The script above will change the default multiplexed port 443 to the default turnserver one if multiplexing is not enabled.

Reply:
I can change it back but it would be more logical to do it the other way around, i.e. have the script change from the default TURNS port (`5349`) when multiplexing _is enabled_. I can put that change in the PR as well if needed.

Reply:
Currently, the default of installing packages is to enable multiplexing and until we switch that default is 443.

Reply:
Changed back in https://github.com/jitsi/jitsi-meet/pull/6172/commits/308e47d42a43fb6d25f0a8be33ea8fcdc6c696e0